modules/packages/Zarr.chpl

```diff
@@ -165,7 +165,7 @@ module Zarr {
 
 
   /* Returns the domain of the `chunkIndex`-th chunk for chunks of size `chunkShape` */
-  private proc getChunkDomain(chunkShape: ?dimCount*int, chunkIndex: dimCount*int) {
+  proc getChunkDomain(chunkShape: ?dimCount*int, chunkIndex: dimCount*int) {
```

Comment:
If `getChunkDomain` is going to be public, consider expanding the docs similar to some of the other functions